.github/workflows/PR_summary.yml

```
@@ -46,10 +46,29 @@ jobs:
         sudo apt-get install -y jq
         # If you have additional dependencies, install them here
 
-    - name: Get changed files
+    - name: Get changed and removed/renamed files
       run: |
         git fetch origin ${{ github.base_ref }}  # fetch the base branch
-        git diff --name-only origin/${{ github.base_ref }}... > changed_files.txt  # get the list of changed files
+        # Get the list of all changed files.
+        git diff --name-only origin/${{ github.base_ref }}... > changed_files.txt
+        # Get all files which were removed or renamed.
+        movedFiles=$(git diff --name-only --diff-filter DR origin/${{ github.base_ref }}...)
+        if [ -n "$movedFiles" ]; then
+          echo "info: found removed file(s) $movedFiles"
+        fi
+        echo $movedFiles > removed_renamed_files.txt
```

```suggestion
        echo "$movedFiles" > removed_renamed_files.txt
```
If this stays, then the quotes should be present.  I'm a little surprised that actionlint did not complain.